docs/MockFunctions.md

```diff
@@ -114,13 +114,17 @@ function that's not directly being tested.
 
 Still, there are cases where it's useful to go beyond the ability to specify
 return values and full-on replace the implementation of a mock function. This
-can be done with the `mockImplementation` method on mock functions:
+can be done with `mockImplementation` or `mockImplementationOnce` methods
+on mock functions:
 
 ```javascript
 var myObj = {
-  myMethod: jest.genMockFunction().mockImplementation(function() {
+  myMethod: jest.genMockFunction().mockImplementationOnce(function() {
     // do something stateful
     return this;
+  }).mockImplementation(function() {
```

Comment:
No, I meant it once. But you are right, it's better to provide two examples.

Currently, if there is a specific implementation defined by `mockImplementationOnce` available then it will be used, otherwise the "default" implementation defined by `mockImplementation` is used. The logic is similar to `mockReturnValueOnce/mockReturnValue` logic.